rts/motoko-rts/src/lib.rs

```diff
@@ -7,7 +7,8 @@
     proc_macro_hygiene,
     // // We do not need simd but this flag enables `core::arch:wasm64`.
     // // See https://github.com/rust-lang/rust/issues/90599
-    simd_wasm64
+    simd_wasm64,
```

Comment:
Is this needed?
_Note_: There is a SIMD-`Cargo.toml` in the `lib/rustlib/src/rust/library/portable-simd` of the nightly